app/controllers/concerns/uber_command.rb

```diff
@@ -51,32 +51,22 @@ def estimate(user_request)
 
     start_addr, end_addr = parse_start_and_end_address(user_request)
 
-    product_id = get_default_product_id_for_lat_lng(origin_lat, origin_lng)
+    product_id = UberAPI.get_default_product_id_for_lat_lng(origin_lat, origin_lng)
```

Comment:
This command as written won't work. Right now this method is being delegated to the `UberComandFormatter` module, where it passes the bearer token. I think that method should probably be extracted and put into UberCommand.rb since it's not presentation related. But it *does* include Rails caching, which should definitely *not* live in `UberAPI`.

Ideally we'd want UberAPI to be abstractable as a gem.